lib/skate_web/endpoint.ex

```diff
@@ -46,14 +48,14 @@ defmodule SkateWeb.Endpoint do
 
   # callback for runtime configuration
   def init(:supervisor, config) do
-    secret_key_base = System.get_env("SECRET_KEY_BASE")
-
     config =
-      if secret_key_base do
+      try do
+        secret_key_base = SecretsManager.fetch!("SECRET_KEY_BASE")
```

Comment:
Shouldn't this be loaded into the application config by `Application.runtime_config()`? Can we replace this whole `try`/`rescue` and `SecretsManager.fetch` with an `Application.get_env`?

Reply:
We are intentionally not using `Application.runtime_config()` for this. For dev and test we don't care about the values, so we have them defined in those environment config files so it's one less thing you have to worry about in your `.env` file.

Reply:
I think you could accomplish that with
`dev.exs`:
```
secret_key_base: "local_secret_key_base_at_least_64_bytes_________________________________"
```
`prod.exs`:
```
secret_key_base: {:secret, "SECRET_KEY_BASE"}
```

Reply:
Updated this.